src/metabase/models/cache_config.clj

```diff
@@ -0,0 +1,17 @@
+(ns metabase.models.cache-config
+  "A model representing cache configuration."
+  (:require
+   [metabase.models.interface :as mi]
+   [methodical.core :as methodical]
+   [toucan2.core :as t2]))
+
+(doto :model/CacheConfig
+  (derive :metabase/model)
+  (derive :hook/updated-at-timestamped?))
```

Comment:
```suggestion
  (derive :hook/timestamped?))
```
this will handle both created_at and updated_at

Reply:
nice, done